### DISCUSSION TITLE
...Where is my stuff?

### DISCUSSION THREAD

**Discussion by anischenk0** [beetbox/beets#4977] [General]
Hello! Sorry about asking stuff here, not sure if it's the right place, but I just ran beets, and now my main partition (which is not where I wanted my stuff!) is about 40 gigs short and the files are nowhere to be found. Could you please help me? My conf file (~/.config/beets) goes:

` directory: /big/Music/sorted
 library: /big/Music
 import:
    move: yes
 replace:
  '[\\/]': _
     '^\.': _
     '[\x00-\x1f]': _
     '[<>:"\?\*\|]': _
     '\.$': _
     '\s+$': ''
     '^\s+': ''
     '^-': _
 asciify_paths: yes 
`
Nothing is to be found in any of the directories specified. I ran "sudo beet import /big/Music/unroot", which is where most stuff is. I only tried '$ beet move' afterwards, which outputs:

`configuration error: /home/w/.config/beets/config.yaml could not be read: mapping values are not allowed here
  in "/home/w/.config/beets/config.yaml", line 7, column 10
`
Thanks!

**Comment by wisp3rwind** [comment]
Duplicate of #4979
